spec/solr/params_spec.rb

```diff
@@ -11,14 +11,14 @@
       expect(params.to_h).to include(fq: ['animals:(dogs\ \+\ cats)'])
     end
 
-    it 'properly handles and escapes multiple values and defaults to OR-ing them together' do
+    it 'properly handles and escapes multiple values and defaults to match function' do
```

Comment:
```suggestion
    it 'properly handles and escapes multiple values and defaults to OR-ing them together with equals comparison' do
```
Maybe something like that?